### ISSUE TITLE
404 object module logs on first click of each API

### ISSUE THREAD

**Issue by alexrosenfeld10** [domaindrivendev/Swashbuckle.AspNetCore#1993]
I'm seeing an issue on version 6.0.0. Steps:

1. Launch the swagger ui. The application logs:

```
[12:10:27  INF] Request Method=GET. Path=/. StatusCode=302. took 34.436 ms
[12:10:27  INF] Request Method=GET. Path=/swagger/index.html. StatusCode=200. took 78.717 ms
[12:10:27  INF] Request Method=GET. Path=/swagger/swagger-ui.css. StatusCode=304. took 5.251 ms
[12:10:27  INF] Request Method=GET. Path=/swagger/swagger-ui-bundle.js. StatusCode=304. took 0.129 ms
[12:10:27  INF] Request Method=GET. Path=/swagger/swagger-ui-standalone-preset.js. StatusCode=304. took 0.407 ms
[12:10:28  INF] Request Method=GET. Path=/v1/swagger/swagger.json. StatusCode=200. took 168.698 ms
```

2. Click on an API to expand it and read details, try it out, etc. Upon doing so, the application logs:

```
[12:10:30  INF] Request Method=GET. Path=/swagger/%5Bobject%20Module%5D. StatusCode=404. took 41.071 ms
```

This 404 log only happens on the first time the API is expanded.

It seems like there is a related issue here: https://github.com/tiangolo/fastapi/issues/2632

**Comment by domaindrivendev** [comment]
This is an upstream issue with swagger - https://github.com/swagger-api/swagger-ui/issues/6850.

I believe it's fixed in their latest version `3.42.0`, and am working on getting SB `6.0.1` out with that version embedded.

**Comment by alexrosenfeld10** [comment]
Thanks @domaindrivendev
